light-chat/index.html

```diff
@@ -1,50 +1,53 @@
-    <head>
-        <meta charset='UTF-8'/>
-        <meta content='width=device-width, initial-scale=1.0' name='viewport'/>
-        <title>JS-Waku light chat</title>
-        <link rel="stylesheet" href="./style.css">
-    </head>
-
-    <body>
-        <div class="content">
-            <div class="header">
-                <h3>Status: <span id='status'></span></h3>
-
-                <details>
-                    <summary>Peer's information</summary>
-
-                    <h4>Content topic</h4>
-                    <p id='contentTopic'></p>
-
-                    <h4>Local Peer Id</h4>
-                    <p id='localPeerId'></p>
-
-                    <h4>Remote Peer Id</h4>
-                    <p id='remotePeerId'></p>
-
-                    <h4>Remote peer's multiaddr</h4>
-                    <p id='remoteMultiAddr'></p>
-                </details>
-            </div>
-
-            <div id="messages"></div>
-
-            <div class="footer">
-                <div class="inputArea">
-                    <input type="text" id="nickText" placeholder="Nickname" />
-                    <textarea id="messageText" placeholder="Message"></textarea>
-                </div>
-    
-                <div class="controls">
-                    <button id="send">Send</button>
-                    <button id="exit">Exit chat</button>
-                </div>
-            </div>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+    <title>JS-Waku light chat</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="apple-touch-icon" href="./favicon.png" />
+    <link rel="manifest" href="./manifest.json" />
+    <link rel="icon" href="./favicon.ico" />
+  </head>
+
+  <body>
+    <div class="content">
+      <div class="header">
+        <h3>Status: <span id="status"></span></h3>
+
+        <details>
+          <summary>Peer's information</summary>
+
+          <h4>Content topic</h4>
+          <p id="contentTopic"></p>
+
+          <h4>Local Peer Id</h4>
+          <p id="localPeerId"></p>
+
+          <h4>Remote Peer Id</h4>
+          <p id="remotePeerId"></p>
+
+          <h4>Remote peer's multiaddr</h4>
+          <p id="remoteMultiAddr"></p>
+        </details>
+      </div>
+
+      <div id="messages"></div>
+
+      <div class="footer">
+        <div class="inputArea">
+          <input type="text" id="nickText" placeholder="Nickname" />
+          <textarea id="messageText" placeholder="Message"></textarea>
         </div>
 
-        <script src="//cdn.jsdelivr.net/npm/protobufjs@7.X.X/dist/protobuf.min.js"></script>
-        <script type='module' src="./index.js"></script>
-    </body>
+        <div class="controls">
+          <button id="send">Send</button>
+          <button id="exit">Exit chat</button>
+        </div>
+      </div>
+    </div>
+
+    <script src="//cdn.jsdelivr.net/npm/protobufjs@7.X.X/dist/protobuf.min.js"></script>
```

Comment:
unrelated to the scope of this PR; curious why there's a `//` prefixed to the source URL?

Reply:
It is called relative protocol and will use protocol of current URL
https://en.wikipedia.org/wiki/URL#prurl

Reply:
Interesting!